src/components/TilePicker.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import * as db from "../firebase";
+import { useForm } from "react-hook-form";
+
+export function TilePicker(props: {
+  tileName: string,
+  tileInfo: db.TileInfo,
+  map: db.MapRepr,
```

Comment:
I wonder if we should emit an event up to Admin rather than passing all this data here... Admittedly this would be inconsistent with how Resizer and TileEditor work.